app/components/SliderQuestions.tsx

```diff
@@ -0,0 +1,37 @@
+import React from 'react';
+import RangeSlider from '~/components/RangeSlider';
+
+interface SliderQuestionsProps {
+    symptom: string;
+    questions: {
+        text: string;
+        answers: string[];
+    }[];
+    onSliderChange: (value: number) => void;
```

Comment:
the same object and onsliderchange is passed to each question - in practice this means each slider is changing a global value and not their own individual value.